forte/base_classes/active_space_method.h

```diff
@@ -139,8 +139,9 @@ class ActiveSpaceMethod {
     /// Useful to get the 3-RDM contribution of fully contracted term of two 2-body operators:
     /// \sum_{puvwxyzστθ} v_{pwxy} t_{uvpz} <Ψ| xσ^+ yτ^+ wτ zθ^+ vθ uσ |Ψ>
     virtual std::vector<double>
-    compute_complementary_H2caa_overlap(const std::vector<size_t>& roots, ambit::Tensor Tbra,
-                                        ambit::Tensor Tket) {
+    compute_complementary_H2caa_overlap([[maybe_unused]] const std::vector<size_t>& roots,
```

Comment:
I notice we used `maybe_unused` here while commented out the parameters in other places. I'd say we should just remove the names here.

Reply:
I think that `maybe_unused` is not the best way to solve this issue. I opted to remove it but keep the names. I like to keep names because it makes the input of the function easier to understand.